### ISSUE TITLE
One column of keys not registering

### ISSUE THREAD

**Issue by coberscheid** [pqrs-org/Karabiner-Elements#2715]
I have a weird issue that appears to be new, perhaps related zu Big Sur 11.3. 

I'm using a German (QWERTZ) IBM Model M with my 2020 MacBook Pro. I've mapped caps_lock to left_option and left_option (Alt) to left_command. This has worked for years over several generations of MacBooks and MacOS.

The first full three key columns from the left are "1qay", "2wsx" and "3edc". With Karabiner-Elements active, these keys work as expected without any modifier, with the left_shift modifier, and with the left_option modifier. With left_shift and left_option combined, the whole second column "2wsx" does not register. The MacOS keyboard viewer shows left_shift and left_command active, but none of the key presses of that column. All other keys work just fine with these two modifiers.

When I quit Karabiner-Elements, these keys "2wsx" with left_shift and left_option (Alt on the physical keyboard) combined do register as expected. Restart Karabiner-Elements - gone.

Since I use left_shift + left_option + w quite often, I'm positive that this is a very new thing. This was not an issue until a few days ago. The only major change I see in that timeframe is upgrading to Big Sur 11.3.

If there's anything I can try on my end, I'd be grateful for hints.

**Comment by mdbraber** [comment]
Did you ever find a way to fix this?

**Comment by coberscheid** [comment]
Yes ... by muscle memory, since right_option (Alt Gr unmapped) works together with left_shift. The original issue persists.
